sql/size_utils.sql

```diff
@@ -0,0 +1,167 @@
+
+-- This file contains utility functions to get the relation size
+-- of hypertables, chunks, and indexes on hypertables.
+
+-- Get relation size of hypertable
+-- like pg_relation_size(hypertable)
+-- (https://www.postgresql.org/docs/9.6/static/functions-admin.html#FUNCTIONS-ADMIN-DBSIZE)
+--
+-- main_table - hypertable to get size of
+--
+-- Returns:
+-- table_size        - Disk space used by main_table (like pg_relation_size(main_table))
```

Comment:
Can we also add index sizes and toast sizes explicitly. Something like the first query here: https://wiki.postgresql.org/wiki/Disk_Usage. I would also bunch the pretty output together and the non-pretty ones together. So you get pretty, pretty, pretty, bytes, bytes, bytes. Instead of pretty, bytes, pretty, bytes. I think thats easier to read.